alacritty/src/config/ui_config.rs

```diff
@@ -473,11 +474,102 @@ pub struct HintBinding {
 /// Hint mouse highlighting.
 #[derive(ConfigDeserialize, Default, Copy, Clone, Debug, PartialEq, Eq)]
 pub struct HintMouse {
-    /// Hint mouse highlighting availability.
-    pub enabled: bool,
-
     /// Required mouse modifiers for hint highlighting.
     pub mods: ModsWrapper,
+
+    /// Mouse button which triggers the command. None to disable mouse hints
```

Comment:
This isn't the documentation pullled for manpages, so it's not expected to match it. I don't think pointing out the purpose of individual variants like `None` here provides any value.

```suggestion
    /// Mouse button which triggers the command.
```